### DISCUSSION TITLE
小白请求

### DISCUSSION THREAD

**Discussion by dittocafe** [wangchucheng/hugo-eureka#75] [General]
非编程人，按照说明搭建了博客，hugo+pages，但模板按照文档第一关就没过去，samplesites里面，config\_defalt\ 文件夹中没有 config.toml文件啊

**Comment by wangchucheng** [comment]
应有config.yaml文件，hugo能处理yaml，toml，json三种类型的文件。


________________________________
发件人: dittocafe <notifications@github.com>
发送时间: Sunday, February 28, 2021 2:23:44 PM
收件人: wangchucheng/hugo-eureka <hugo-eureka@noreply.github.com>
抄送: Subscribed <subscribed@noreply.github.com>
主题: [wangchucheng/hugo-eureka] 小白请求 (#75)


非编程人，按照说明搭建了博客，hugo+pages，但模板按照文档第一关就没过去，samplesites里面，config_defalt\ 文件夹中没有 config.toml文件啊

―

**Reply by dittocafe** [reply]
多谢。我再试试，有点晕。go1.16 装上了，初始化了，config.yaml还没改成功。

**Reply by dittocafe** [reply]
这个模板确实挺漂亮，但超出了我的照猫画虎能力。。。
